### ISSUE TITLE
Add ace self-interaction to disable headcams

### ISSUE THREAD

**Issue by m0kka** [Special-Operations-Brigade/SOB-Issue-Tracker#258]
Not sure what this would implicitly compell people to do.......

**Comment by oliv3lrg** [comment]
I don't think this is required, maybe a CBA setting instead for users of our mod who don't want them?

**Comment by gumbekk** [comment]
I think it's fine as an ACE Interaction, also stops people from turning in on only while downed
